docs/src/extend/firmware-management/building-image/rugpi/tutorial.md

```diff
@@ -48,92 +70,103 @@ Feel free to clone the project if you want to make your own customizations, howe
     cd tedge-rugpi-image
     ```
 
-2. Install [just](https://just.systems/man/en/chapter_5.html) which is used to run different tasks provided by the project
+2. Create a custom `.env` file which will be used to store secrets
 
     ```sh
-    curl --proto '=https' --tlsv1.2 -sSf https://just.systems/install.sh | sudo bash -s -- --to /usr/bin
+    cp env.template .env
     ```
 
-    Check out the [justfile website](https://just.systems/man/en/chapter_5.html) for more installation options.
-
-### Customizing your own image
-
-1. Fork the project
-
-2. Edit the authorized ssh keys for each profile
+    The `.env` file will not be committed to the repo
 
-    Edit the `root_authorized_keys` under each toml file in the `profiles/` directory, and add the public ssh keys whom you want to grant ssh access to for the device. Below shows one example.
+3. Add your public ssh key to the `.env` file
 
-    ```toml title="file: profiles/default.toml"
-    #...
+    Adding public SSH keys to the project will allow you to access after the image is flashed to the device (which is critical for the onboarding process). The SSH keys are provided in the form of environment variables where the variable names start with `SSH_KEY_<description>`.
 
-    [parameters.ssh]
-    root_authorized_keys = """
-    ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDfhQGWWw73ponAokdNSRZ5cQc9/CIX1TLQgYlr+BtObKoO4UNFP1YSbgK03GjhjeUid+QPmV+UURqxQTqLQoYWqUFP2CYkILFccVPmTvx9HLwupI+6QQKWfMDx9Djfph9GzInymaA5fT7hKppqittFrC/l3lkKgKTX5ohEOGshIbRgtgOYIaW3ByTx3urnaBbYCIgOyOZzSIyS0dUkwsiLu3XjPspgmn3Fs/+vofT/yhBe1carW0UM3ivV0JFfJzrxbCl/F7I2qwfjZXsypjkwlpNupUMuo3xPMi8YvNvyEu4d+IEAqO1dCcdGcxlkiHxrdITIpVLt5mjJ2LauHE/H bootstrap
-    """
+    For example:
 
-    # ...
+    ```
+    SSH_KEYS_bootstrap="ssh-rsa xxxxxxx"
+    SSH_KEYS_user2="ssh-rsa xxxxxxx"
+    SSH_KEYS_user3="ssh-rsa xxxxxxx"
```

Comment:
Though I want to make sure people know how to add ssh keys...but yes the "user" part sounds like it might be related to users rather than adding a key to the root user.

Would `SSH_KEYS_key1` be better?

Reply:
I removed the additional keys, and extended the sentence to also mention where these keys are added to (e.g. the root user authorized_keys).

Resolved by 7270cdc6049d33d2f049f24b61836e4eadaaa984